libkbfs/config_local.go

```diff
@@ -1459,3 +1460,19 @@ func (c *ConfigLocal) SetKBFSService(k *KBFSService) {
 	}
 	c.kbfsService = k
 }
+
+// RootNodeWrappers implements the Config interface for ConfigLocal.
+func (c *ConfigLocal) RootNodeWrappers() []func(Node) Node {
+	c.lock.RLock()
+	defer c.lock.RUnlock()
+	ret := make([]func(Node) Node, len(c.rootNodeWrappers))
+	copy(ret, c.rootNodeWrappers)
+	return ret
```

Comment:
I wonder if you can instead do `return c.rootNodeWrappers[:]`, since the underlying `func(Node) Node` elements aren't going to change.